src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -28,16 +30,22 @@ internal class BindingWorkflow
     {
         private readonly IHost host;
         private readonly ProjectInformation project;
-        private readonly IProjectSystemHelper projectSystemHelper;
+        private readonly IProjectSystemHelper projectSystem;
+        private readonly IProjectSystemFilter projectFilter;
         private readonly SolutionBindingOperation solutionBindingOperation;
 
-        internal readonly Dictionary<string, RuleSetGroup> LanguageToGroupMapping = new Dictionary<string, RuleSetGroup>
+        internal readonly Dictionary<Language, RuleSetGroup> LanguageToGroupMapping = new Dictionary<Language, RuleSetGroup>
         {
-            {SonarQubeServiceWrapper.CSharpLanguage, RuleSetGroup.CSharp },
-            {SonarQubeServiceWrapper.VBLanguage, RuleSetGroup.VB }
+            {Language.CSharp, RuleSetGroup.CSharp },
+            {Language.VBNET, RuleSetGroup.VB }
         };        
 
         public BindingWorkflow(IHost host, ProjectInformation project)
+            : this(host, project, null)
+        {
+        }
+
+        internal /*for testing purposes*/ BindingWorkflow(IHost host, ProjectInformation project, IProjectSystemFilter projectFilter)
```

Comment:
Return to an internal ctor for `IProjectSystemFilter`; not using the 'local services' approach since the filter is local only to the binding workflow and is potentially different per workflow run.